Mathlib/Algebra/Quotient.lean

```diff
@@ -0,0 +1,61 @@
+/-
+Copyright (c) 2021 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+-/
+import Mathlib.Mathport.Rename
+/-!
+# Algebraic quotients
+
+This file defines notation for algebraic quotients, e.g. quotient groups `G ⧸ H`,
+quotient modules `M ⧸ N` and ideal quotients `R ⧸ I`.
+
+The actual quotient structures are defined in the following files:
```

Comment:
Can you add a porting note saying these should be changed? Or maybe change them directly to the future file, I don't know.